Investigate flaky UI test plotter should disable case button

Onprem builds fails quite often with this error;

```
FAILED unit_tests/gui/test_main_window.py::test_that_gui_plotter_disables_add_case_button_when_no_data - assert not True
 +  where True = <built-in method isEnabled of QToolButton object at 0x7f17214448b0>()
 +    where <built-in method isEnabled of QToolButton object at 0x7f17214448b0> = <PyQt5.QtWidgets.QToolButton object at 0x7f17214448b0>.isEnabled
= 1 failed, 1686 passed, 70 skipped, 1 xfailed, 319 warnings in 677.00s (0:11:16) =
Error: Process completed with exit code 1.
```

https://github.com/equinor/komodo-releases/actions/runs/6985809451/job/19011232025

--- a/src/ert/gui/tools/plot/plot_case_selection_widget.py
+++ b/src/ert/gui/tools/plot/plot_case_selection_widget.py
@@ -25,6 +25,7 @@ class CaseSelectionWidget(QWidget):
         self.__add_case_button.setToolButtonStyle(Qt.ToolButtonTextBesideIcon)
         self.__add_case_button.setText("Add case to plot")
         self.__add_case_button.setIcon(QIcon("img:add_circle_outlined.svg"))
+        self.__add_case_button.setEnabled(len(self._cases) > 0)
         self.__add_case_button.clicked.connect(self.addCaseSelector)
 
         add_button_layout.addStretch()

